src/sentry/static/sentry/app/components/events/interfaces/spans/dragManager.tsx

```diff
@@ -211,12 +372,27 @@ class DragManager extends React.Component<DragManagerProps, DragManagerState> {
     const childrenProps = {
       isDragging: this.state.isDragging,
 
+      // left handle
+
       onLeftHandleDragStart: this.onLeftHandleDragStart,
       leftHandlePosition: this.state.leftHandlePosition,
-      viewWindowStart: this.state.viewWindowStart,
+
+      // right handle
 
       onRightHandleDragStart: this.onRightHandleDragStart,
       rightHandlePosition: this.state.rightHandlePosition,
+
+      // custom window selection
```

Comment:
How is a dragged window more custom than moving sliders around? They both seem like ways to control the same window but with different control types.

Reply:
This UX addition was one of the feedback from alpha customers; and one that didn't make the cut on the initial implementation (due to bandwidth constraints).

IMO, it's an alternative/additional option to create the selection.